src/app/child-dev-project/children/children-list/children-list.component.ts

```diff
@@ -78,8 +70,8 @@ export class ChildrenListComponent implements OnInit, AfterViewInit {
         "age",
         "schoolClass",
         "schoolId",
-        "attendance-school",
-        "attendance-coaching",
+        "school",
```

Comment:
these columns are still about attendance, aren't they? the new naming doesn't make that clear.

Reply:
This is done to align it with the `id`s that were used in the configuration for the columns (and is in this case the value which is passed to the attendance-component). Once #512 is done, a filter can be generated just by providing the id of that column.